### DISCUSSION TITLE
ekran koruyucu

### DISCUSSION THREAD

**Discussion by lagerthaa88** [rocksdanister/lively#2371] [Q&A]
ekran koruyucuyu yapmak  istiyorum ama kurulum gerekli diyor ne yapmam lazım

**Comment by HylaireHub** [comment]
https://github.com/rocksdanister/lively/wiki/Screen-Saver Burada anlatıyor detaylıca. Linkten indirmen lazım. https://github.com/rocksdanister/lively/releases/download/v2.1.0.6/lively_screen_saver.zip Bu dosyayı ayıklayıp içindeki .scr dosyasını Windows klasörüne atıp üzerine sağ tıklayınca çıkan yükle butonuna basman lazım. Ondan sonra Başlat Menüsü'nden ekran koruyucuyu değiştire tıklayıp Lively'i seçmen lazım. Süresini de ayarladıktan sonra çalıştığını göreceksin.
